plasmapy/physics/__init__.py

```diff
@@ -1,34 +1,8 @@
 # 'physics' is a tentative name for this subpackage.  Another
 # possibility is 'plasma'.  The organization is to be decided by v0.1.
 
-from .parameters import (Alfven_speed,
-                         ion_sound_speed,
-                         thermal_speed,
-                         kappa_thermal_speed,
-                         gyrofrequency,
-                         gyroradius,
-                         plasma_frequency,
-                         Debye_length,
-                         Debye_number,
-                         inertial_length,
-                         magnetic_pressure,
-                         magnetic_energy_density,
-                         upper_hybrid_frequency,
-                         lower_hybrid_frequency,
-                         )
-
-from .quantum import (deBroglie_wavelength,
-                      Fermi_energy,
-                      Thomas_Fermi_length,
-                      )
-
-from .relativity import Lorentz_factor
-
-from .transport import Coulomb_logarithm
-
-from .distribution import (Maxwellian_1D,
-                           Maxwellian_speed_1D,
-                           Maxwellian_velocity_3D,
-                           Maxwellian_speed_3D,
-                           kappa_velocity_1D,
-                           kappa_velocity_3D)
+from .parameters import *
+from .quantum import *
+from .relativity import *
+from .transport import *
+from .distribution import *
```

Comment:
We've had problems with forgetting to put stuff into __init__ before, so I love this solution!